src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/TextEditorContextMenu.cs

```diff
@@ -420,6 +420,18 @@ private static ITextPointer GetContentPosition(ITextPointer position)
         // Default ContextMenu for TextBox and RichTextBox.
         private class EditorContextMenu : ContextMenu
         {
+            public EditorContextMenu() : base()
+            {
+                if(ThemeColorization.IsFluentWindowsThemeEnabled)
+                {
+                    SetResourceReference(StyleProperty, typeof(ContextMenu));
```

Comment:
EditorContextMenu is private. It cannot be accessed in XAML.

Reply:
As it inherits from ContextMenu why is this code even required then?

Reply:
As of now what happens is, this class picks up the Aero2 contextmenu theme. I suspect that since Aero2 is still the default, and Win11 theme is applied above that, it still picks up the Aero2 theme

Reply:
Perhaps we should have the proper theme selection rather than merging dictionaries for the next round of reviews to avoid issues like this.